{{cookiecutter.project_name}}/{{cookiecutter.package_name}}/__init__.py

```diff
@@ -1 +1,18 @@
-"""{{cookiecutter.package_name}}."""
+from pathlib import Path
+from typing import Union
+
+
+def get_version() -> tuple:
```

Comment:
suggestion (non-blocking): Let's explain why we need this function here.

"""
For conda packaging / other reason, we need a means to obtain the version from the package from a safe place.
"""

Reply:
My motivation for including this is to have `__version__` in the format I expect - I use this a lot, and it is consistent with the rest of the Brightway libraries.

On some level it could be considered weird to have this function everywhere, but at the same time it doesn't really hurt anything, and is easy to test.